packages/plugins/whisper-geth/src/whisperGethClient.js

```diff
@@ -1,17 +1,16 @@
 import { __ } from 'embark-i18n';
 const async = require('async');
-const GethMiner = require('./miner');
 const semver = require('semver');
 const constants = require('embark-core/constants');
 
 const DEFAULTS = {
   "BIN": "geth",
-  "VERSIONS_SUPPORTED": ">=1.9.7",
+  "VERSIONS_SUPPORTED": ">=1.8.14",
```

Comment:
@emizzle what's the reason for dropping the officially supported minimum version back to `1.8.14`, i.e. as opposed to keeping it matched up with `1.9.7` in [packages/plugins/geth/src/gethClient.js#L12](https://github.com/embark-framework/embark/blob/refactor/whisper-node-arch/packages/plugins/geth/src/gethClient.js#L12)?